conf/machine/raspberrypi3-64.inc

```diff
@@ -6,12 +6,7 @@
 
 DISTRO_ARCH ?= "arm64"
 
-# Raspberry Pi needs non-free-firmware repository.
-DISTRO_APT_SOURCES:append = " conf/distro/emlinux-bookworm-full.list"
```

Comment:
Keep this DISTRO_APT_SOURCES:append in the first commit, otherwise 3B+ cannot work when the commit is checked out.

Reply:
I fixed it. 
`raspberrypi3-64.inc` contains `DISTRO_APT_SOURCES:append` and `IMAGE_FSTYPES` only in the first commit.